components/tikv_util/src/file.rs

```diff
@@ -153,8 +154,17 @@ mod tests {
         delete_file_if_exist(&non_existent_file).unwrap();
     }
 
+    fn gen_ascii_chars(size: usize) -> String {
+        let mut s = String::new();
+        let mut rng = thread_rng();
+        for _ in 0..size {
+            s.push((rng.next_u32() % 256) as u8 as char);
+        }
+        s
+    }
+
     fn gen_rand_file<P: AsRef<Path>>(path: P, size: usize) -> u32 {
-        let s: String = thread_rng().gen_ascii_chars().take(size).collect();
+        let s: String = gen_ascii_chars(size);
```

Comment:
Maybe https://docs.rs/rand/0.6.5/rand/distributions/struct.Alphanumeric.html is enough

Reply:
Thx! I will fix it.